common/src/main/java/dev/latvian/mods/kubejs/item/ItemModificationProperties.java

```diff
@@ -57,4 +82,85 @@ public void setFoodProperties(Consumer<FoodBuilder> consumer) {
 		consumer.accept(builder);
 		item.setFoodPropertiesKJS(builder.build());
 	}
+
+	private Multimap<Attribute, AttributeModifier> mutateAttributeMap() {
+		Multimap<Attribute, AttributeModifier> attributes = null;
+		if (item instanceof DiggerItem diggerItem) {
+			diggerItem.defaultModifiers = ArrayListMultimap.create(diggerItem.defaultModifiers);
+			attributes = diggerItem.defaultModifiers;
+		} else if (item instanceof SwordItem swordItem) {
+			swordItem.defaultModifiers = ArrayListMultimap.create(swordItem.defaultModifiers);
+			attributes = swordItem.defaultModifiers;
+		} else if (item instanceof TridentItem tridentItem) {
+			tridentItem.defaultModifiers = ArrayListMultimap.create(tridentItem.defaultModifiers);
+			attributes = tridentItem.defaultModifiers;
+		} else if (item instanceof ArmorItem armorItem) {
+			armorItem.defaultModifiers = ArrayListMultimap.create(armorItem.defaultModifiers);
+			attributes = armorItem.defaultModifiers;
```

Comment:
This could probably be abstracted to an interface (such as `ModifiableItemKJS` injected to these classes via Mixin, couldn't it?

Reply:
Classes are now mixined with ModifiableItemKJS.